babel.config.js

```diff
@@ -27,7 +27,7 @@ module.exports = function babelConfig(api) {
     "lodash"
   ];
   if (api.env("development")) {
-    plugins.push("react-hot-loader/babel");
+    plugins.push(["react-hot-loader/babel", { safetyNet: false }]);
```

Comment:
This fixes [the console error](https://github.com/nextstrain/auspice/pull/1520#discussion_r985327588) in `auspice develop`, but it seems weird and I don't understand why it's necessary only after the webpack upgrades (see 6b9fddf0e187ee5bf9f2bd9537eb0eee164156d2).

Reply:
🤷 JS imports / require is messy business

Reply:
😬

Reply:
Hot reloading is working well now 👍